theano/sandbox/cuda/tests/test_dnn.py

```diff
@@ -42,13 +42,30 @@ def test_dnn_conv_desc_merge():
     # CDataType is not DeepCopyable so this will crash if we don't use
     # borrow=True
     f = theano.function([], [theano.Out(desc1, borrow=True),
-                             theano.Out(desc2, borrow=True)])
+                             theano.Out(desc2, borrow=True)],
+                        mode=mode_with_gpu)
 
     d1, d2 = f()
 
     # This will be the case if they are merged, which would be bad.
     assert d1 != d2
 
+    desc1v2 = dnn.GpuDnnConvDesc(border_mode='valid', subsample=(2, 2),
+                                 conv_mode='conv')(img_shp, kern_shp)
+    f = theano.function([], [theano.Out(desc1, borrow=True),
+                             theano.Out(desc1v2, borrow=True)],
+                        mode=mode_with_gpu)
+    assert len(f.maker.fgraph.apply_nodes)
```

Comment:
Is the goal of this line really to check that the graph has more than 0 apply node?